### DISCUSSION TITLE
Does the Ibis table asof_join work with the DuckDB backend?

### DISCUSSION THREAD

**Discussion by derekmahar** [ibis-project/ibis#6482] [Q&A] [answered]
Does the [Ibis](https://ibis-project.org/) table [`asof_join`](https://ibis-project.org/api/expressions/tables/#ibis.expr.types.relations.Table.asof_join) function work with the [DuckDB](https://duckdb.org/) backend?  Does it use the SQL [`ASOF JOIN`](https://duckdb.org/docs/sql/query_syntax/from#as-of-joins) operator that DuckDB [0.8.0](https://github.com/duckdb/duckdb/releases/tag/v0.8.0) introduced?

**Comment by cpcloud** [comment, marked as answer]
Hi @derekmahar 👋🏻, and welcome to ibis!

We have not yet implemented `asof_join` for the duckdb backend, but it's on our near-term roadmap. I'm not sure if it'll make the 6.0 release, but it may make the next one (6.1).

I'm going to convert this to a feature request issue and mark the discussion resolved!

**Reply by cpcloud** [reply]
https://github.com/ibis-project/ibis/issues/6487
